### ISSUE TITLE
Use subpixel rendering to improve text crispness

### ISSUE THREAD

**Issue by RickoNoNo3** [fyne-io/fyne#5347]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

Fyne rendered a blurred text on Windows. And this poor rendering is not affected by DPI settings.

### How to reproduce

Run Demo directly

### Screenshots

125% DPI

![image](https://github.com/user-attachments/assets/93455dfa-4f14-4fd2-91c6-1120ecd1b4d6)

100% DPI

![image](https://github.com/user-attachments/assets/9112c7e6-ed8b-49c8-aa85-e20e334306f6)

### Example code

`go run fyne.io/fyne/v2/cmd/hello`

or

`go run fyne.io/fyne/v2/cmd/fyne_demo`

### Fyne version

2.5.3

### Go compiler version

1.21.3

### Operating system and version

Windows 10

**Comment by dweymouth** [comment]
This doesn't look excessively blurry to me beyond the fact that Fyne (or rather the go-text dependency) doesn't currently use subpixel rendering when drawing text. Would be nice to have but I wouldn't consider it a "bug", rather a missing feature

**Comment by andydotxyz** [comment]
Yes, if I zoom in on those images you can see that the "edge blur" on the Fyne is almost the same as the native text.

**Comment by andydotxyz** [comment]
I believe that this was resolved with recent text changes

**Comment by dweymouth** [comment]
This is technically a wontfix since it was about subpixel rendering to improve all text crispness on lowDPI, not about the bug that affected only certain lines of text being stretched
